test/testsuite/lantest_io_monitor.h

```diff
@@ -26,7 +27,7 @@
 #endif
 
 /* Global Debug flag - controlled by -b option in lantest.c */
-extern uint8_t Debug;
+extern _Bool Debug;
```

Comment:
How come you use `_Bool` here and `bool` data types elsewhere?

Reply:
Well spotted. Which do you prefer? _Bool is more explicit (pre C99), but it should be bool everywhere as we target C11. I'll update

Reply:
I have a mild preference for the `bool` keyword. :)